json-endpoints/jetpack/class.jetpack-json-api-get-term-taxonomy-backup-endpoint.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+class Jetpack_JSON_API_Get_Term_Taxonomy_Backup_Endpoint extends Jetpack_JSON_API_Endpoint {
+	// /sites/%s/term-taxonomies/backup      -> $blog_id
```

Comment:
Good point! Yes, this isn't useful to include in a backups interface. Removed in 1d2d420. The terms endpoint returns plenty.